Rocket.Chat/Info.plist

```diff
@@ -106,194 +108,194 @@
-                <string>AppIcon-76</string>
-                <string>AppIcon-83.5</string>
-            </array>
-            <key>UIPrerenderedIcon</key>
-            <false/>
-        </dict>
-        <key>CFBundleAlternateIcons</key>
-        <dict>
-            <key>Black</key>
-            <dict>
-                <key>CFBundleIconFiles</key>
-                <array>
-                    <string>Black-29</string>
-                    <string>Black-40</string>
-                    <string>Black-76</string>
-                    <string>Black-83.5</string>
-                </array>
-                <key>UIPrerenderedIcon</key>
-                <false/>
-            </dict>
-            <key>Red</key>
-            <dict>
-                <key>CFBundleIconFiles</key>
-                <array>
-                    <string>Red-29</string>
-                    <string>Red-40</string>
-                    <string>Red-76</string>
-                    <string>Red-83.5</string>
-                </array>
-                <key>UIPrerenderedIcon</key>
-                <false/>
-            </dict>
-            <key>BnW</key>
-            <dict>
-                <key>CFBundleIconFiles</key>
-                <array>
-                    <string>BnW-29</string>
-                    <string>BnW-40</string>
-                    <string>BnW-76</string>
-                    <string>BnW-83.5</string>
-                </array>
-                <key>UIPrerenderedIcon</key>
-                <false/>
-            </dict>
-            <key>Grey</key>
-            <dict>
-                <key>CFBundleIconFiles</key>
-                <array>
-                    <string>Grey-29</string>
-                    <string>Grey-40</string>
-                    <string>Grey-76</string>
-                    <string>Grey-83.5</string>
-                </array>
-                <key>UIPrerenderedIcon</key>
-                <false/>
-            </dict>
-            <key>White</key>
-            <dict>
-                <key>CFBundleIconFiles</key>
-                <array>
-                    <string>White-29</string>
-                    <string>White-40</string>
-                    <string>White-76</string>
-                    <string>White-83.5</string>
-                </array>
-                <key>UIPrerenderedIcon</key>
-                <false/>
-            </dict>
-            <key>Blue</key>
-            <dict>
-                <key>CFBundleIconFiles</key>
-                <array>
-                    <string>Blue-29</string>
-                    <string>Blue-40</string>
-                    <string>Blue-76</string>
-                    <string>Blue-83.5</string>
-                </array>
-                <key>UIPrerenderedIcon</key>
-                <false/>
-            </dict>
-        </dict>
-    </dict>
+	<key>CFBundleIcons</key>
+	<dict>
+		<key>CFBundlePrimaryIcon</key>
+		<dict>
+			<key>CFBundleIconFiles</key>
+			<array>
```

Comment:
🤷‍♂️

Reply:
Happens all the time with .plist files 😬